CHANGES.md

```diff
@@ -1,9 +1,11 @@
-### Unreleased
+### 1.0.1 (2020-02-12)
 
 - Add support for an `ALCOTEST_COLOR={auto,always,never}` environment variable
   to control the colorization of terminal output. (#209, @mjambon)
+- Fix handling of asynchronous exceptions in `Alcotest_{async,lwt}`. (#212,
```

Comment:
It's not "asynchronous exceptions", but just "exceptions" in general that were fixed.

Reply:
Thanks. See https://github.com/mirage/alcotest/pull/214/files.